runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -353,7 +362,11 @@ sub loadData {
 	my $logName          = "$logPath/loadData-W${workloadNum}I${appInstanceNum}-$time.log";
 	my $namespace = $self->appInstance->namespace;
 	
-	$logger->debug("loadData for workload $workloadNum, appInstance $appInstanceNum");
+	#Setting outputted workloadNum to empty string if only one workload exists
+	my $workloadCount = $self->workloadDriver->{workloadCount};
+	$workloadNum = $workloadCount > 1 ? "Workload $workloadNum," : "";
+
+	$logger->debug("loadData for $workloadNum appInstance $appInstanceNum");
```

Comment:
The word 'workload' will only appear if there's more than one workload because it's embedded within the $workloadNum variable. Otherwise it's omitted because the workload is already implied (feel free to let me know if you think I should change this).